src/main/resources/db/migration/V1_0__INITIALIZE.sql

```diff
@@ -4,7 +4,7 @@ create table USER
     FIRST_NAME   varchar(50) not null,
     LAST_NAME    varchar(50) not null,
     BIRTH_DATE   datetime    not null,
-    EMAIL        varchar(50) not null,
+    EMAIL        varchar(50) not null unique ,
```

Comment:
so now everybody will have to drop the data base in order to run the project. The main idea behind flyway is: we don't need to drop table each time we do a fix or change. So the right way would be: 1) create new file, call it some how v1_NUM_USER_EMAIL_FIX.sql 2) alter table